packages/host/tests/acceptance/interact-submode-test.gts

```diff
@@ -50,17 +54,21 @@ module('Acceptance | interact submode tests', function (hooks) {
   setupServerSentEvents(hooks);
   setupOnSave(hooks);
   setupWindowMock(hooks);
-  setupMatrixServiceMock(hooks);
-  setupSessionsServiceMock(hooks);
+  setupMatrixServiceMock(hooks, () => realmPermissions);
 
   hooks.afterEach(async function () {
-    window.localStorage.removeItem('recent-cards');
-    window.localStorage.removeItem('recent-files');
+    windowMock.localStorage.removeItem('recent-cards');
+    windowMock.localStorage.removeItem('recent-files');
+    // intentionally not using the window mock because otherwise the actual
+    // app's local storage stomps on top of the window mock
+    window.localStorage.removeItem('boxel-session');
```

Comment:
The way the library is designed to be used is to import `window` where it’s going to be used, then it’s the real one in development/production and the mock on in tests.

Reply:
so i think the problem is that in the `resources/realm.ts` I'm not importing the mock. I'll add that import there